packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -82,35 +79,32 @@ const createIndexHeaderTitleDarkStyles = css({
  * Create index modal.
  */
 function CreateIndexModal({
-  toggleIsVisible,
-  resetForm,
   isVisible,
   namespace,
   error,
   clearError,
   inProgress,
   createIndex,
+  closeCreateIndexModal,
   ...props
 }: React.ComponentProps<typeof CreateIndexForm> & {
-  toggleIsVisible: (isVisible: boolean) => void;
-  resetForm: () => void;
   isVisible: boolean;
   namespace: string;
   error: string | null;
   clearError: () => void;
   inProgress: boolean;
   createIndex: () => void;
```

Comment:
nit: should we clear previous error when user clicks on create index button?


https://user-images.githubusercontent.com/1305718/187974176-a3e13103-45d1-4420-a266-3cd8dcdf9571.mov

Reply:
Nice catch! 👀